Tests/ClientTests/CreditCard/CreditCardInputFieldHelperTests.swift

```diff
@@ -0,0 +1,78 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+
+@testable import Client
+
+class CreditCardInputFieldHelperTests: XCTestCase {
```

Comment:
The coverage is high in this class but there are some guards and if conditions (edge cases) which are perfect for unit testing